src/subsplit_dag.cpp

```diff
@@ -322,8 +326,8 @@ std::vector<Bitset> SubsplitDAG::GetSortedVectorOfNodeBitsets() const {
 std::vector<Bitset> SubsplitDAG::GetSortedVectorOfEdgeBitsets() const {
   std::vector<Bitset> edges;
   for (auto i : storage_.GetLines()) {
-    auto parent_bitset = GetDAGNode(i.GetParent()).GetBitset();
-    auto child_bitset = GetDAGNode(i.GetChild()).GetBitset();
+    auto parent_bitset = i.GetParentNode().GetBitset();
```

Comment:
I let this slide on the first subsplit DAG PR refactor, but I really do prefer more informative variable names everywhere, including in situations like this.